docs/spark/how-to-guides/dotnet-spark-jupyter-notebooks.md

```diff
@@ -0,0 +1,108 @@
+
+See the [getting started tutorial](../tutorials/get-started.md) for more information on setting up your .NET for Apache Spark environment.
+
+## Prepare environment
+
+To work with Jupyter Notebooks, you'll need two things.
+
+1. Install [.NET Interactive global .NET tool](https://github.com/dotnet/interactive/blob/main/docs/NotebooksLocalExperience.md)
+1. Download `Microsoft.Spark` NuGet package.
+    1. Navigate to the [Microsoft.Spark](https://www.nuget.org/packages/Microsoft.Spark/) NuGet package page.
+
+        > [!IMPORTANT]
+        > By default, the latest version of the package is downloaded. **Make sure that the version you download is the same as your Apache Spark .NET Worker.**
+
+    1. In the **Info** pane, select **Download package**. Doing so downloads the latest version of the package to a file with a name similar to  *microsoft.spark.[PACKAGE-VERSION].nupkg*.
+    1. Unzip the recently downloaded package. The unzipped directory should contain a subdirectory called *jars*. Take note of the path since it's used at a later time.
+
+## Start .NET for Apache Spark
+
+Start .NET for Apache Spark in debug mode. This will will set up a process that will wait for connections from a [SparkSession](xref:Microsoft.Spark.Sql.SparkSession). Make sure to provide the path to the jar for the respective version of Apache Spark you're using.
+
+```bash
+spark-submit \
+  --class org.apache.spark.deploy.dotnet.DotnetRunner \
+  --master local \
+  <path-to-microsoft-spark-jar> \
+  debug
+```
+
+## Create notebook
+
+You can use different interfaces to interact with Jupyter. For a browser-based interface, use Jupyter Notebooks or Jupyter Lab. For a local editor experience, use VS Code.
+
+### Jupyter Notebooks & Jupyter lab
+
+1. In another command prompt, start Jupyter Notebook or Jupyter Lab use one of the commands below:
+
+    **Jupyter Notebook**
+
+    ```bash
+    jupyter notebook
+    ```
+
+    **Jupyter Lab**
+
+    ```bash
+    jupyter lab
+    ```
+
+1. Create a new .NET Interactive notebook.
+1. When the notebook opens, write and execute your code in the notebook cells.
+
+### Visual Studio Code (preview)
+
+> [!IMPORTANT]
+> To use Jupyter Notebooks in VS Code, you have to install:
+>
+>- [VS Code Insiders](https://code.visualstudio.com/insiders/)
+>- [.NET Interactive Notebooks extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode)
+
+1. Open VS Code
+1. Open the command palette **View > Command Palette**
+
+    The command palette appears. Enter the following command to create a new .NET Interactive notebook:
+
+    ```text
+    .NET Interactive: Create new blank notebook
+    ```
+
+    Alternatively, if you want to open an existing .NET Interactive notebook with the *.ipynb* extension, use the following command:
+
+    ```text
+    .NET Interactive: Open notebook
+    ```
+
+1. When the notebook opens, write and execute your code in the notebook cells.
```

Comment:
Ditto for here. I think it would be good to show through examples what this would look like.

Reply:
Done